### PR TITLE
Update/Delete

### PR DESCRIPTION
**Pull request by Beltarixxx**
Update/Delete

### CODE REVIEW

File: CriminalsSystem/Controller/MainController.cs [hunk 1]
```diff
@@ -41,6 +41,32 @@ public void AddPrisoner(Prisoner prisoner)
...
+            using (PrisonersDBEntities model = new PrisonersDBEntities())
+            {
+                var deletePrisoner = model.Prisoners.Where(pr => pr.Id == id).FirstOrDefault();
+                if (deletePrisoner != null)
+                {
+                    model.Prisoners.Remove(deletePrisoner);
+                    model.SaveChanges();
+                }
+            }
+        }
```
Comment (stankopk) [inline CriminalsSystem/Controller/MainController.cs:55]: Add an empty row, please

File: CriminalsSystem/Controller/MainController.cs [hunk 2]
```diff
@@ -41,6 +41,32 @@ public void AddPrisoner(Prisoner prisoner)
...
+        internal void UpdatePrisoner(int id, Prisoner prisoner)
```
Comment (stankopk) [inline CriminalsSystem/Controller/MainController.cs:56]: Why internal ?! Use public, please.

File: CriminalsSystem/View/MainForm.cs [hunk 3]
```diff
@@ -40,13 +40,52 @@ private void RefreshTable()
...
+            using (PrisonersDBEntities model = new PrisonersDBEntities())
+            {
+                var deletePrisoner = model.Prisoners.Where(pr => pr.Id == id).FirstOrDefault();
+                if (deletePrisoner != null)
+                {
+                    model.Prisoners.Remove(deletePrisoner);
+                    model.SaveChanges();
+                }
+            }
+        }
```
Comment (stankopk) [inline CriminalsSystem/View/MainForm.cs:54]: same here

File: CriminalsSystem/View/MainForm.cs [hunk 4]
```diff
@@ -40,13 +40,52 @@ private void RefreshTable()
...
+        internal void UpdatePrisoner(int id, Prisoner prisoner)
```
Comment (stankopk) [inline CriminalsSystem/View/MainForm.cs:55]: same here

**Review by stankopk — CHANGES_REQUESTED** [review]
See my comments and do the changes, please.

**Review by WIKSATA1337 — APPROVED** [review]
nice

**Pull request merged**